superbench/common/utils/gen_traffic_pattern_config.py

```diff
@@ -178,5 +179,14 @@ def gen_traffic_pattern_host_group(host_list, pattern):
         )
     else:
         logger.error('Unsupported traffic pattern: {}'.format(pattern.type))
-    host_group = __convert_config_to_host_group(config, host_list)
-    return host_group
+    host_groups = __convert_config_to_host_group(config, host_list)
+    # write traffic pattern host groups to specified path
+    if pattern_config_path:
+        with open(pattern_config_path, 'a') as f:
+            f.write('pattern_type: {}'.format(pattern.type) + '\n')
+            for serial_index, host_group in enumerate(host_groups):
+                for parallel_index, host_list in enumerate(host_group):
+                    exec_index = '{}_{}'.format(serial_index, parallel_index)
+                    f.write("{}: '{}' ".format(exec_index, ' '.join(host_list)))
```

Comment:
exec_index seems useless, can you keep the same format as current ib validation file? "," to concat different hosts in each group and ";" to concat parallel groups

Reply:
And single line for each run

Reply:
updated for both